Explain where data need to be stored in module's context (#1131)
PrestaShop/docs: modules/creation/good-practices.md

- A merchant is likely to have a shop running on a different shop than yours. In case a module is using PHP extensions not installed by default by PHP,
add a preliminary check before using them (I.e with `extension_loaded`). This prevents fatal errors to be thrown on shops on which these extensions aren’t enabled.

> - About files, we also recommend you to save them inside a `/var/modules/YOUR_MODULE/` directory. Putting them outside the `/modules` directory will help for scalable applications, and also to keep data if needed in case of you uninstall the module.

Author: ```suggestion
- About files, if you plan to save customer data generated by your module, we also recommend you to save them inside a `/var/modules/YOUR_MODULE/` directory. Putting them outside the `/modules` directory will help for scalable applications, and also to keep data if needed in case of you uninstall the module.
```

Author: @kpodemski @PululuK is it better?

Reviewer: ```suggestion
- If you need to store static files: we recommend putting temporary or cache files in prestashop's `/var/cache/<env>/modules/YOUR_MODULE/` directory, and static or shared files in prestashop's  `/var/modules/YOUR_MODULE/` directory. Writing in the `/modules` directory is not recommended, because doing so will make it harder to use your module in distributed environments. In addition, by storing files outside your module's directory, they are kept even if the module is uninstalled.
```

> - About files, we also recommend you to save them inside a `/var/modules/YOUR_MODULE/` directory. Putting them outside the `/modules` directory will help for scallable applications, and also to keep data if needed in case of you uninstall the module.

Reviewer 2: ```suggestion
- About files, we also recommend you to save them inside a `/var/modules/YOUR_MODULE/` directory. Putting them outside the `/modules` directory will help for scalable applications, and also to keep data if needed in case of you uninstall the module.
```

> 

Reviewer 3: to avoid confusion, I think it might be interesting to specify in which context this recommendation is valid, because in some cases storing data outside the module maybe a bad idea too

Reviewer 2: That's a good point.
